mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -3699,16 +3727,55 @@ static func::FuncOp createCpuAttentionKernelWithMlir(ModuleOp module,
   auto qkMaxs = createOpAndInfer<tosa::ReduceMaxOp>(
       builder, loc, cast<ShapedType>(qkTensor.getType()).getElementType(),
       qkTensor, reductionAxis);
-  auto normilizedQkTensor = createOpAndInfer<tosa::SubOp>(
+  auto normalizedQkTensor = createOpAndInfer<tosa::SubOp>(
       builder, loc, cast<ShapedType>(qkTensor.getType()).getElementType(),
       qkTensor, qkMaxs);
   auto expsTensor = createOpAndInfer<tosa::ExpOp>(
       builder, loc,
-      cast<ShapedType>(normilizedQkTensor.getType()).getElementType(),
-      normilizedQkTensor);
+      cast<ShapedType>(normalizedQkTensor.getType()).getElementType(),
+      normalizedQkTensor);
   auto expsSums = createOpAndInfer<tosa::ReduceSumOp>(
       builder, loc, cast<ShapedType>(expsTensor.getType()).getElementType(),
       expsTensor, reductionAxis);
+  Value lseTensor;
+  if (returnLSE) {
```

Comment:
Just put a comment up there it is doing, it would help
`(log(sum_exp) + m) / ln(2)`